modules/farm_fd2_examples/src/entrypoints/soil_disturbance/App.vue

```diff
@@ -0,0 +1,270 @@
+            id="set-speed-button"
+            data-cy="set-speed-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeSpeed"
+          >
+            Speed = 3
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>Set Passes</td>
+        <td>
+          <BButton
+            id="set-passes-button"
+            data-cy="set-passes-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changePasses"
+          >
+            Passes = 4
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>includePasses</td>
+        <td>
+          <BFormCheckbox
+            id="include-passes-checkbox"
+            data-cy="include-passes-checkbox"
+            switch
+            v-model="form.includePasses"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>Add Planter</td>
+        <td>
+          <BButton
+            id="add-equipment-button"
+            data-cy="add-equipment-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="addEquipment"
+          >
+            Add Planter
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>Clear</td>
+        <td>
+          <BButton
+            id="clear-equipment-button"
+            data-cy="clear-equipment-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="clearEquipment"
+          >
+            Clear Equipment
+          </BButton>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h5>Component Event Payloads</h5>
+  <table class="example-table">
+    <thead>
+      <th>Event</th>
+      <th>Payload</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td>valid</td>
+        <td>{{ validity.isValid }}</td>
+      </tr>
+      <tr>
+        <td>area</td>
+        <td>{{ form.area }}</td>
+      </tr>
+      <tr>
+        <td>depth</td>
+        <td>{{ form.depth }}</td>
+      </tr>
+      <tr>
+        <td>equipment</td>
+        <td>{{ form.equipment }}</td>
+      </tr>
+      <tr>
+        <td>include passes</td>
+        <td>{{ form.includePasses }}</td>
+      </tr>
+      <tr>
+        <td>speed</td>
+        <td>{{ form.speed }}</td>
+      </tr>
+      <tr>
+        <td>passes</td>
+        <td>{{ form.passes }}</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <div
+    data-cy="page-loaded"
+    v-show="false"
+  >
+    {{ pageDoneLoading }}
+  </div>
+</template>
+
+<script>
+import SoilDisturbance from '@comps/SoilDisturbance/SoilDisturbance.vue';
+import { BButton } from 'bootstrap-vue-next';
+
+export default {
+  components: {
+    SoilDisturbance,
+    BButton,
+  },
+  data() {
+    return {
+      required: true,
+      form: {
+        includePasses: true,
+        area: 100,
+        depth: 0,
+        equipment: [],
+        passes: 1,
+        speed: 0,
+      },
+      validity: {
+        showStyling: false,
```

Comment:
Add `soilDisturbance` here and `v-bind` above to `valid` event from the `SoilDisturbance` component.